run.py

```diff
@@ -97,6 +110,8 @@ def config_validator():
     parser = argparse.ArgumentParser()
     parser.add_argument('-t', '--testmode', action='store_true',
                         help='Run server in "test mode".')
+    parser.add_argument('--backup', action='store_true',
+                        help='backup existing issues.db and update db schema and issues dump.')
```

Comment:
One minor thing: `update db schema and issues dump` -- This should just be a command to create a backup, not do any schema updates.